### PR TITLE
Initial ruby rules

### PR DESCRIPTION
**Pull request by j-mcnally**

### CODE REVIEW

**Comment by johnkoht** [comment]
:+1:

File: ruby.md [hunk 1]
```diff
@@ -1 +1,118 @@
...
+my_sweet_variable = some_method("hi")
+```
+
+
+## Line Length
+
+Code lines should break every 80 characters.
+Code broken into additional lines should be indented under the main line of the code statement.
+
+Strings broken accross lines should be indented so all text is stacked in the same columns.
```
Comment (andimal) [inline ruby.md]: across

**Comment by andimal** [comment]
looks good other than the typo

**Comment by j-mcnally** [comment]
Word will clean up and merge @johnkoht you good with this?

**Comment by j-mcnally** [comment]
Going to let @johnkoht review and then hopefully merge!

**Comment by johnkoht** [comment]
:+1:

**Comment by lrdiv** [comment]
:+1:

**Pull request merged**